### PR TITLE
Implement Gripper

### LINKED ISSUE #3: Implement Gripper
Poseidon has a pneumatic gripper using one double ended piston.

The pneumatic setup for the gripper is unique. Instead of controlling the position with a one double solenoid, it has two double solenoid used to change the pressure in the piston when extended. (low pressure for cubes, high pressure for cones)

Two double solenoids means there are 4 possible states for the gripper subsystem. Open, low pressure, high pressure and an unknown state.

Tasks:

- [x] Create a GripperSubsystem.java
- [x] Use two of the DoubleSolenoid objects ([here](https://docs.wpilib.org/en/stable/docs/software/hardware-apis/pneumatics/pneumatics.html#double-solenoids-in-wpilib))
- [x] Create a command to control the GripperSubsystem
- [x] Create a more advanced command to activate the solenoids for 0.5 seconds and then turn them off. (to avoid having them electrically active all the time since they don't need to be).

### PR DESCRIPTION
**Pull request by ryanidk**
Implements a subsystem and commands to control the gripper. Also binds those commands to the following buttons:

X - turn off both solenoids electronically (total stop command)
Y - no pressure
A - low pressure
B - high pressure

Closes #3.

### CODE REVIEW

**Review by ErikCald — APPROVED** [review]
Looks amazing!

File: networktables.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (weizhang-2706) [inline networktables.json:1]: Do we need this?

File: simgui-ds.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{
```
Comment (weizhang-2706) [inline simgui-ds.json:1]: Do we need this?

File: simgui.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (weizhang-2706) [inline simgui.json:1]: Do we need this?

**Review by weizhang-2706 — APPROVED** [review]

**Pull request merged**